### ISSUE TITLE
[ISSUE]: in F0 Det only crepe_tiny and rmpve_onnx

### ISSUE THREAD

**Issue by Adstodus** [w-okada/voice-changer#1242]
### Voice Changer Version

MMVCServerSIO_win_onnxdirectML-cuda_v.1.5.3.18a.zip

### Operational System

Windows 10

### GPU

Radeon RX 590 Fatboy 8GB GDDR5

### Read carefully and check the options

- [X] I've tried to Clear Settings
- [ ] Sample/Default Models are working
- [X] I've tried to change the Chunk Size
- [X] GUI was successfully launched
- [ ] I've read the [tutorial](https://github.com/w-okada/voice-changer/blob/master/tutorials/tutorial_rvc_en_latest.md)
- [ ] I've tried to extract to another folder (or re-extract) the .zip file

### Model Type

ONNX

### Issue Description

having choppy voice and its using my cpu instead of my gpu, i can only use crepe_tiny and rmvpe_onnx, saw that some problems can be fixed by using dio but all other options including dio have (N/A) so i cant use them, trying to speak makes it sound like there is a massive mosquito constantly flying next to my ear

### Application Screenshot

![image](https://github.com/w-okada/voice-changer/assets/156482970/37976674-b39b-49af-8f28-e194ee44c568)

### Logs on console

![image](https://github.com/w-okada/voice-changer/assets/156482970/a7c2975e-0bb2-4d82-90df-b4cf32656b5e)

**Comment by KuukoShan** [comment]
Not sure about some of your issues but I noticed in the screenshot that the tune is set to 0. You should set it to a positive number if you are going from male to female voice and a negative number for female to male. If the desired voice is of the same gender as yours then you may only need to move it slightly to the left or right. But leaving it at 0 will sound weird usually. It may help you on fixing how bad it sounds.

I'm regards to your other issues, check here as there are people with the same issues as yours so may be able to find a solution there.

**Comment by Adstodus** [comment]
yeah, i know how it works but its hard to adjust when i cant even hear it working because how much it lags

**Comment by KuukoShan** [comment]
I'd set the IN setting to 1.0 instead of 0.1 as that's a really low volume.
I would also recommend to read the tutorial so you can understand better the different types of F0 and which one you should use: 

https://github.com/w-okada/voice-changer/blob/master/tutorials/tutorial_rvc_en_latest.md#f0-det-f0-extractor
